### DISCUSSION TITLE
Stop line diagnostics on hover?

### DISCUSSION THREAD

**Discussion by olimorris** [VonHeikemen/lsp-zero.nvim#146] [Q&A]
It's only started bothering me today but does anyone know how to stop line diagnostics when you hover over a line?

![Screen Shot 2023-02-06 at 17 54 28](https://user-images.githubusercontent.com/9512444/217048136-3588748a-5a7a-43de-91e1-0a210be02dcf.gif)

**Comment by VonHeikemen** [comment]
lsp-zero just creates a keybinding to show the diagnostics, it doesn't do it automatically. You probably have a `CursorHold` autocommand somewhere in your config.
